### DISCUSSION TITLE
Errors in log (International dates not supported)

### DISCUSSION THREAD

**Discussion by croldan72** [moralmunky/Home-Assistant-Mail-And-Packages#523] [General]
I'm getting the following errors in the logs, pretty numerously:

"INFO (SyncWorker_4)[custom_components.mail_and_packages.helpers] International dates not supported. (time data 'Sunday, October 3' does not match format '%A, %B %d')"

They seem to be info messages and I guess I should just ignore. Just trying to determine if there is anyway for me to prevent.

**Comment by firstof9** [comment]
Turn down your logging to `warning`, the messages are safe to ignore.
